### PR TITLE
Fixes #6964: The time synchronization technique doesn't work on AIX

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/6964

### CODE REVIEW

File: techniques/systemSettings/misc/clockConfiguration/3.1/metadata.xml [hunk 1]
```diff
@@ -16,6 +16,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
 <TECHNIQUE name="Time settings">
+  <DEPRECATED>This technique version has been superseded by a new version. It will no longer be available in the next stable version of Rudder. Please upgrade to the latest version.</DEPRECATED>
```
Comment (peckpeck) [inline techniques/systemSettings/misc/clockConfiguration/3.1/metadata.xml]: We shouldn't deprecate a version in a minor release.
This is a bug in rudder-dev

**Comment by peckpeck** [comment]
LGTM

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**